### PR TITLE
Wildcard subscription doc

### PR DESCRIPTION
**Pull request by Meggielqk**
I drafted the Wildcard Subscription page. Please help to review, especially the demo part. @wivwiv , @tigercl

### CODE REVIEW

File: en_US/messaging/mqtt-shared-subscription.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ You can use client tools to try this messaging service in EMQX. This section int
 
 The following procedure demonstrates how to form groups for multiple subscribers that can share the subscription to the same topic and how these subscribers will receive the messages from the shared subscription.
 
-In this demonstration, you can create one client connection `demo` as a publisher to publish messages on the topic `t/1`. Then, you can create three client connections as subscribers, such as `Subscriber1`, `Subscriber2` and `Subscriber3`.  The subscribers can be divided into groups `a` and `b`, and both groups subscribe to the topic `t/1`.
+In this demonstration, you can create one client connection `Demo` as a publisher to publish messages on the topic `t/1`. Then, you can create three client connections as subscribers, such as `Subscriber1`, `Subscriber2` and `Subscriber3`.  The subscribers can be divided into groups `a` and `b`, and both groups subscribe to the topic `t/1`.
```
Comment (tigercl) [inline en_US/messaging/mqtt-shared-subscription.md:18]: The Client ID in the screenshot is demo?

File: en_US/messaging/mqtt-wildcard-subscription.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+`#` (U+0023) is a wildcard character that matches any number of levels within a topic. When using a multi-level wildcard, it must occupy an entire level and must be the last character of the topic, for example:
+
+```pgsql
+"#" is valid, matches all topics
+"sensor/#" is valid
+"sensor/bedroom#" is invalid (+ or # are only used as a wildcard level)
+"sensor/#/temperature" is invalid (# must be the last level)
+```
+
+## Try Wildcard  Subscription with MQTTX Client
```
Comment (tigercl) [inline en_US/messaging/mqtt-wildcard-subscription.md]:
```suggestion
## Try Wildcard Subscription with MQTTX Client
```

File: en_US/messaging/mqtt-wildcard-subscription.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+This section demonstrates how to use the MQTTX client to simulate subscriptions to wildcard topics.
```
Comment (tigercl) [inline en_US/messaging/mqtt-wildcard-subscription.md]:
```suggestion
This section demonstrates how to use the MQTTX client to create subscriptions to wildcard topics. 
```

File: en_US/messaging/mqtt-wildcard-subscription.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+"sensor/#" is valid
+"sensor/bedroom#" is invalid (+ or # are only used as a wildcard level)
+"sensor/#/temperature" is invalid (# must be the last level)
+```
+
+## Try Wildcard  Subscription with MQTTX Client
+
+This section demonstrates how to use the MQTTX client to simulate subscriptions to wildcard topics. 
+
+In this demonstration, you can create one client connection `demo` as a publisher to publish messages. Then, you can create another client connection as a subscriber. The subscriber subscribes to the following wildcard topics:
```
Comment (tigercl) [inline en_US/messaging/mqtt-wildcard-subscription.md]: The two client IDs in the screenshot, one with the first letter capitalized and one without.

File: en_US/messaging/mqtt-wildcard-subscription.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+   <img src="/Users/emqx/Documents/GitHub/emqx-docs/en_US/messaging/assets/Configure-new-connection-general.png" alt="Configure-new-connection-general" style="zoom:35%;" />
+
+2. Click **+** in the **Connections** pane to create another connection as a subscriber. Set **Name** to `Subscriber`.
+
+3. Select `Subscriber` in **Connections** and click **+ New Subscription**. In the pop-up dialogue, input `testtopic/+/temperature` in the **Topic** field. Leave other options as default.
+
+   <img src="./assets/wildcard-sub-1.png" alt="wildcard-sub-1" style="zoom: 38%;" />
+
+4. Select `Demo` in **Connections**. Type `testtopic/room1/temperature` in the topic field and type the message payload `28 degree` in the message field. Click the send button. Send another message with the topic `testtopic/room2/temperature`.
```
Comment (tigercl) [inline en_US/messaging/mqtt-wildcard-subscription.md]:
```suggestion
4. Select `Demo` in **Connections**. Type `testtopic/room1/temperature` in the topic field and type the message payload `28 degree` in the message field. Click the send button. Send another message with the same payload to the topic `testtopic/room2/temperature`.
```

File: en_US/messaging/mqtt-wildcard-subscription.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+      <img src="./assets/wildcard-sub-2.png" alt="wildcard-sub-2" style="zoom:40%;" />
+
+5. Select `Subscriber` in **Connections**. You can see the subscriber receives the two messages the publisher sends.
```
Comment (tigercl) [inline en_US/messaging/mqtt-wildcard-subscription.md]:
```suggestion
5. Select `Subscriber` in **Connections**. You can see the subscriber receives the two messages sent by the publisher.
```

File: en_US/messaging/mqtt-shared-subscription.md (hunk 1 above)
Comment (Meggielqk) [reply en_US/messaging/mqtt-shared-subscription.md:18]: Changed to "Demo" and updated the screenshots.

File: en_US/messaging/mqtt-wildcard-subscription.md (hunk 4 above)
Comment (Meggielqk) [reply en_US/messaging/mqtt-wildcard-subscription.md]: Changed to "Demo" throughout.

File: dir.yaml [hunk 7]
```diff
@@ -142,6 +142,10 @@
       title_cn: 共享订阅
       path: messaging/mqtt-shared-subscription
       lang: en
+    - title_en: Wildcard Subscription
+      title_cn: 通配符订阅
+      path: messaging/mqtt-wildcard-subscription
+      lang: en
```
Comment (wivwiv) [inline dir.yaml]: 我看已经有中文了
```suggestion
```

**Review by wivwiv — CHANGES_REQUESTED** [review]

File: dir.yaml (hunk 7 above)
Comment (Meggielqk) [reply dir.yaml]: Removed “en”.

**Review by wivwiv — APPROVED** [review]

**Pull request merged**